internal/spicedbx/schema.go

```diff
@@ -45,10 +45,10 @@ definition PREFIX/tenant {
 	permission edit = editor + parent_tenant->edit_child_tenants
 	permission delete = deleter + parent_tenant->delete_child_tenants
 
-	relation tenant_viewer: role#actor
-	relation tenant_editor: role#actor
-	relation tenant_creator: role#actor
-	relation tenant_deleter: role#actor
+	relation tenant_viewer: foo/role#actor
+	relation tenant_editor: foo/role#actor
+	relation tenant_creator: foo/role#actor
+	relation tenant_deleter: foo/role#actor
```

Comment:
Should these be `PREFIX/role` rather than `foo/role`?

Reply:
lmao, I commited the wrong thing. Fixed.